processor/src/kitspaceYaml.ts

```diff
@@ -0,0 +1,100 @@
+import { promises as fs } from 'fs'
+import jsYaml from 'js-yaml'
+import log from 'loglevel'
+import path from 'node:path'
+import { z } from 'zod'
```

Comment:
```suggestion
import fs from 'node:fs/promises'
import path from 'node:path'

import { z } from 'zod'
import jsYaml from 'js-yaml'
import log from 'loglevel'
```

Reply:
Ok, I'll use "node:fs/promises".  I'm using tsserver to auto organize imports now so I don't want to apply this as suggested though.

Reply:
[539c8c4](https://github.com/kitspace/kitspace-v2/pull/478/commits/539c8c4cc6732e73a2bd70dc6cab9053bf617370)